packages/protocol/contracts/common/linkedlists/SortedLinkedList.sol

```diff
@@ -71,8 +71,18 @@ library SortedLinkedList {
     bytes32 lesserKey,
```

Comment:
we should probably create a new function, since this makes the worst case worse, and SortedOracles may be hitting the worst case most of the time

Reply:
We could make a separate function anyways, but SortedOracles uses the WithMedian lists, which are not affected by this change.